### PR TITLE
US-2.1 Revised spacecraft movement and supply decrease fix

### LINKED ISSUE #21: Supplies subtract instead of doing the percentage math
Affecting #1 and #7.

### LINKED ISSUE #24: US-2.1 Revised Spacecraft Movement
User Story: US-2.1
Revised Spacecraft Movement
Priority: 1

Story: As a player, I want to be able to use screen buttons to move my spacecraft Up, Down, Right or Left,and specify a distance from my current position to travel using an enumerated input structure so I can navigate around the play area

### PR DESCRIPTION
**Pull request by SimonBartonPSU**
Resolves #21 
Closes #24 

Created a separate function that is called to decrement supplies, which actually decreases supplies by 2% rather than just decrementing by 2. Additionally, the supplies value that is shown on the status menu is just a number of supplies now rather than a percentage.

I was able to test this new behavior quite a bit and I think it's working well. Feel free to try it out and let me know if I'm wrong.

### CODE REVIEW

File: gameObjects.js [hunk 1]
```diff
@@ -127,11 +127,15 @@ var spaceship = {
       } 
     }
   }
+  
+  function supplyDecrease() {
+    spaceship.supplies = Math.floor(spaceship.supplies * 0.98)
```
[the remark this answers is not in the document]
Comment (SimonBartonPSU) [reply gameObjects.js]: Thanks doggy

**Comment by SimonBartonPSU** [comment]
Forgot to mention in last commit that my fix resolves #18

**Review by LiamRotchford5671 — COMMENTED** [review]
Tested all distance values and all four buttons on each distance case. It all seems to be working perfectly with all the master code. I really like how you set the buttons up like a game controller D-pad it looks great! Feel free to merge away.

**Review by LiamRotchford5671 — APPROVED** [review]
Tested all distance values and all four buttons on each distance case. It all seems to be working perfectly with all the master code. I really like how you set the buttons up like a game controller D-pad it looks great! Feel free to merge away. (of course I hit the wrong button the first time lol)

**Pull request merged**